libraries/adaptive-expressions/src/expressionFunctions.ts

```diff
@@ -2988,7 +2983,37 @@ export class ExpressionFunctions {
                         return {value, error};
                     }),
                 ReturnType.Boolean,
-                ExpressionFunctions.validateIsMatch)
+                ExpressionFunctions.validateIsMatch),
+            
+            // Type Checking Functions
+            new ExpressionEvaluator(ExpressionType.isString, ExpressionFunctions.apply(
+                (args: any[]): boolean =>  typeof args[0] === 'string'),
+            ReturnType.Boolean,
+            ExpressionFunctions.validateUnary),
+            new ExpressionEvaluator(ExpressionType.isInteger, ExpressionFunctions.apply(
+                (args: any[]): boolean =>  typeof args[0] === 'number' && args[0] % 1 === 0),
```

Comment:
what's the difference between Number.isInteger and x % 1 !== 0, we can choose one from them